integration/main.go

```diff
@@ -152,13 +153,64 @@ func createTestNamespace(clientset kubernetes.Interface) string {
 
 func deleteTestNamespace(clientset kubernetes.Interface) {
 	name := opts.namespace
-	if err := clientset.Core().Namespaces().Delete(name, &metav1.DeleteOptions{}); err != nil {
+	if err := clientset.CoreV1().Namespaces().Delete(name, &metav1.DeleteOptions{}); err != nil {
 		glog.Fatalf("failed to delete namespace %q (error: %v)", name, err)
 	}
 	glog.Infof("Namespace %v is deleted", name)
 }
 
 func deployIstioCA(clientset kubernetes.Interface) {
+	// Create Role
```

Comment:
why isn’t this a yaml like the other ? (why create in code vs yaml)

Reply:
To me that's a minor question. We can change the code to yaml files easily in the future. The tests are the same. :)

Reply:
sure